spec/support/initializers/vcr.rb

```diff
@@ -15,3 +24,10 @@
     i.request.uri.sub!(/:\/\/.*#{Regexp.escape(u.host)}/, "://#{u.host}")
   end
 end
+
+VCR_OPTIONS = {
+  twitter_fetch_status: {
+    cassette_name: "twitter_fetch_status",
```

Comment:
if we have an outer key called twitter_fetch_status, do we still need to specify the casette name?

Reply:
The outer key is just an arbitrary name (it could be anything), the value is what matters: https://github.com/thepracticaldev/dev.to/pull/6672/files#diff-c0355da51cd64b256ba070012a2c0072R3

I decided to group all VCR options around the code. For now it's a hash of one but I plan to see if I can find more repeated ones and move them there